### ISSUE TITLE
False positive `:docstring-no-summary` on multiline docstrings

### ISSUE THREAD

**Issue by serioga** [clj-kondo/clj-kondo#1507]
**version**

clj-kondo v2021.12.16

**platform**

Windows 10

**problem**

The warning “First line of the docstring should be a capitalized sentence ending with punctuation.” for multiline docstrings like:

```
Initialize UUID from string representation.
Accept only zero-padded representation.
```
**repro**

```clojure
(defn from-string
  "Initialize UUID from string representation.
   Accept only zero-padded representation."
  [_])
```

```clj_kondo_docstring.clj:4:3: warning: First line of the docstring should be a capitalized sentence ending with punctuation.```

**config**

```clojure
:linters {:docstring-no-summary {:level :warning}}
```

**expected behavior**

Expects no warning.

**Comment by borkdude** [comment]
Should be fixed on master, please test.

**Comment by camsaul** [comment]
I'm still running in to this with clj-kondo 2022.01.15

**Comment by borkdude** [comment]
@camsaul Please provide a repro.

**Comment by borkdude** [comment]
@camsaul You might be running into this. Just checking.

The `:docstring-no-summary` linter is very strict. The summary has to be on the first _line_. If the sentence is spread out over multiple lines, then it reports a warning. See discussion here: https://github.com/clj-kondo/clj-kondo/discussions/1516
We could make an extra setting `:single-line true / false`  or so to tweak this.
